src/services/refactors/generateGetAccessorAndSetAccessor.ts

```diff
@@ -186,4 +204,38 @@ namespace ts.refactor.generateGetAccessorAndSetAccessor {
             ? changeTracker.insertNodeAtClassStart(file, <ClassLikeDeclaration>container, accessor)
             : changeTracker.insertNodeAfter(file, declaration, accessor);
     }
+
+    function updateReadonlyPropertyInitializerStatementConstructor(changeTracker: textChanges.ChangeTracker, file: SourceFile, constructor: ConstructorDeclaration, accessorName: AcceptedNameType, fieldName: AcceptedNameType) {
+        if (constructor.body) {
+            const initializerStatement = find(constructor.body.statements, (stmt =>

```

Comment:
An assignment to the private name can happen anywhere really and not just in expressionStatemetns, it can be in a comma expression nested in a call... you want to use `FindAllReferences.getReferenceEntriesForNode` here instead of searching the body yourself. a few notes:
- this could return you nodes outside the body of the constructor, these you want to ignore, so filter out ones that fall outside the pos/end of the constructor
- you only want to look at places where you are writing to the property, use `isWriteAccess(node)` to filter read only operations